src/components/SearchInput/SearchInput.module.scss

```diff
@@ -0,0 +1,27 @@
+.SearchIconContainer {
+  flex: 0 0 auto;
+  position: relative;
+
+  /* TextInput Overrides */
+  > input {
+    height: 40px;
+    border-radius: 3px;
```

Comment:
I've got a follow up task to make some of these shared:
https://app.asana.com/0/1150068311310825/1158592990903755/f